Sources/ManageRegistration.php

```diff
@@ -342,7 +342,7 @@ function ModifyRegistrationSettings($return_config = false)
 		array('check', 'requirePolicyAgreement', 'text_label' => $txt['admin_privacy_policy'], 'value' => 1, 'disabled' => 1),
 		array('warning', empty($policy) ? 'error_no_privacy_policy' : ''),
 	'',
-		array('int', 'coppaAge', 'subtext' => $txt['setting_coppaAge_desc'], 'onchange' => 'checkCoppa();'),
+		array('int', 'coppaAge', 'subtext' => $txt['zero_to_disable'], 'onchange' => 'checkCoppa();'),
```

Comment:
also delete the new string you added since it's no longer needed 👍

Reply:
Which string specifically?

Reply:
@SychO9 if you're referring to the `setting_cppaAge_desc` text string, that has already been removed in a different commit.